cocos/ui/UIListView.cpp

```diff
@@ -504,6 +504,12 @@ void ListView::selectedItemEvent(TouchEventType event)
 void ListView::interceptTouchEvent(TouchEventType event, Widget *sender, Touch* touch)
 {
     ScrollView::interceptTouchEvent(event, sender, touch);
+    
```

Comment:
@TimothyZhang 
Why scrollView doesn't call Layout::interceptTouch to propagate the touch event to its parent?